watchmal/model/classifier.py

```diff
@@ -1,5 +1,12 @@
 import torch.nn as nn
+import torch
 from hydra.utils import instantiate
+from torch_geometric.nn import global_max_pool, global_add_pool, global_mean_pool
+from torch.nn import Sequential, Linear, ReLU, Dropout, BatchNorm1d as BN
```

Comment:
instead of importing all these, we already just `import torch.nn as nn,` so could you remove this and then just use e.g. `nn.Linear` in the code instead of `Linear`, to be consistent with the rest of the code using these same functions?